web/src/app/wizard/utilTestData.js

```diff
@@ -23,6 +23,10 @@ const keys = [
     label: 'Email',
     value: 'email',
   },
+  {
+    label: 'Universal',
+    value: 'universal',
+  },
```

Comment:
can leave this out since it's wizard specific and probably not worth doing the experimental flag check

Reply:
https://github.com/target/goalert/pull/3837